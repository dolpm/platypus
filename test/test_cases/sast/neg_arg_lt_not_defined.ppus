pipe test [x: &'a int] |> &'a int {
  |> x;
}

pipe main [] |> unit {
  |> ();
}
