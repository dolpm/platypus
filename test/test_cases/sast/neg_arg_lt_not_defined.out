Fatal error: exception Failure("lifetime 'a used but not defined in test")
